### DISCUSSION TITLE
Alternative packaging

### DISCUSSION THREAD

**Discussion by daktras420** [fosskers/aura#832] [Ideas]
We have in arch package format tar. Gz for pacman. Red hat based distro use rpm. Debian based distro use Deb. Can your project use the following project or other way to use 7z instead?  I wana make a new distro is why i ask.
https://github.com/pts/pts-tiny-7z-sfx

**Comment by fosskers** [comment]
Aura calls `makepkg` internally to construct packages, and all compression strategies are specified in `/etc/makepkg.conf`. You could try experimenting with settings there and see if that meets your needs.

**Comment by daktras420** [comment]
Thank you. I do some test and checks once I am done getting some parts for
my pc.
